packages/fast-components-react-msft/src/index.ts

```diff
@@ -1,3 +1,5 @@
 import Button from "./button";
```

Comment:
I think you'll need a rebase as there are a few other components that have gone in not listed here.